servers/quarkus-server/src/testFixtures/java/org/projectnessie/server/catalog/AbstractIcebergCatalogTests.java

```diff
@@ -74,7 +74,11 @@ protected RESTCatalog catalog() {
   }
 
   protected Map<String, String> catalogOptions() {
-    return singletonMap(CatalogProperties.WAREHOUSE_LOCATION, WAREHOUSE_NAME);
+    return ImmutableMap.of(
+        CatalogProperties.WAREHOUSE_LOCATION,
+        WAREHOUSE_NAME,
+        "adls.sas-token.account.dfs.core.windows.net",
```

Comment:
Do we need this ADLS-specific option for all tests that inherit from this class?